### ISSUE TITLE
nullable property

### ISSUE THREAD

**Issue by s16508pjwstk** [bjansen/swagger-schema-validator#16]
Hi,

does the validator accepts x-nullable flag for property? If not, is there any possibility to do make property nullable?

**Comment by bjansen** [comment]
No, it's currently not supported. It looks like it's a vendor extension though (prefixed with `x-`).

It's also not supported in either the [JSON Schema Validation](http://json-schema.org/draft/2019-09/json-schema-validation.html) specification, nor in [json-schema-validator](https://github.com/java-json-tools/json-schema-validator) which is used to do the actual validation.

=> I don't think this kind of validation can be done easily.
